templates/events/calendar.html

```diff
@@ -58,20 +58,20 @@ <h1 class="text-center">{{title}}
       <div class="form-check form-check-inline"><h4>Show</h4></div>
       <div class="form-check form-check-inline">
         <input type="checkbox" class="form-check-input" id="materialInline1" {% if "approved" in select_status %}checked{%endif%}>
-        <label class="form-check-label" for="materialInline1">Approved</label>
-      </div>
-      <div class="form-check form-check-inline">
-          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>
-          <label class="form-check-label" for="materialInline2">Disapproved</label>
+        <label class="form-check-label" for="materialInline1">Pending</label>
```

Comment:
Let's use Bootstrap Pill Badges like User events. 
https://github.com/rokwire/events-manager/blob/skip_campus_events_GUI/templates/events/user-events.html#L39
https://getbootstrap.com/docs/4.0/components/badge/#pill-badges. 
Let's use badge-success for Published and badge-warning for Pending. Thanks.